packages/components/src/components/single-select/shadow.tsx

```diff
@@ -670,6 +666,8 @@ export class KolSingleSelect implements SingleSelectAPI {
 	@Watch('_rows')
 	public validateRows(value?: number): void {
 		this.controller.validateRows(value);
+		const rows = value ?? 5;
+		this.host?.style.setProperty('--visible-options', `${rows}`);
```

Comment:
Macht das nicht im Prinzip das gleich wie vorher, nur imperativ statt deklarativ?
In den Snapshots sieht man ja auch, dass weiterhin das Style-Attribut gesetzt wird.

Für tatsächlich dynamische Properties wie diese, würde ich perönlich eine Ausnahme von der "keine inline styles" Regel machen.

@deleonio Wie siehst du das?